lib/openers/sumatra-opener.js

```diff
@@ -9,11 +9,13 @@ export default class SumatraOpener extends Opener {
     const args = [
       '-forward-search',
       `"${texPath}"`,
-      lineNumber,
+      `"${lineNumber}"`,
       `"${filePath}"`
     ]
 
-    child_process.execFile(sumatraPath, args, (error) => {
+    const command = `${sumatraPath} ${args}`
```

Comment:
Unless there's something odd about how Sumatra parses arguments, this will have to be changed to 
~~~ js
const command = `${sumatraPath} ${args.join(' ')}`
~~~
otherwise the arguments will end up being comma-separated since that's the result of `Array#toString()`.
For reference see my commit https://github.com/thomasjo/atom-latex/commit/ea338aaf33218963159021f1d94b05f15631b7d8, but I wanted to give you a chance to fix this. If not I can happily merge this PR and apply my fix afterwards :bow: